src/main/java/com/cst438/controller/AssignmentController.java

```diff
@@ -108,11 +199,23 @@ public List<AssignmentStudentDTO> getStudentAssignments(
             @RequestParam("semester") String semester) {
 
         // TODO remove the following line when done
+        List<Assignment> assignments = assignmentRepository.findByStudentIdAndYearAndSemesterOrderByDueDate(studentId, year, semester);
+        List<AssignmentStudentDTO> AssignmentStudentDTOList = new ArrayList<>();
+        for (Assignment a : assignments) {
+            AssignmentStudentDTOList.add(new AssignmentStudentDTO(
+                    a.getAssignmentId(),
+                    a.getTitle(),
+                    a.getDueDate(),
+                    a.getSection().getCourse().getCourseId(),
+                    a.getSection().getSecId(),
+                    a.getSection().getSectionNo()
```

Comment:
Should be grade, not section no

Reply:
Fixed with commit [8ee9dd7](https://github.com/jdolarcsumb/CST438-Assignment2-02/commit/8ee9dd7028c2f4e36b7f0912a2fbf4ed8110a42b)

Reply:
Fixed via commit [6530f92](https://github.com/jdolarcsumb/CST438-Assignment2-02/commit/6530f922836b5439ec329687e55fe4475881e3e2)